src/hotspot/share/runtime/os.cpp

```diff
@@ -91,6 +100,11 @@ julong os::free_bytes = 0;          // # of bytes freed
 
 static size_t cur_malloc_words = 0;  // current size for MallocMaxTestWords
 
+// CRaC
+static const char* _crengine = NULL;
```

Comment:
Looks like you unintentionally reverted recent changes - search for `_crengine_args` and handling code.

Reply:
Thanks, I will check this